ci/repokitteh/modules/ownerscheck.star

```diff
@@ -152,20 +159,19 @@ def _reconcile(config, specs=None):
   return results
 
 
-def _comment(config, results, force=False):
+def _comment(config, results, assignees, sender, force=False):
```

Comment:
@itayd is this the right way to get at PR author? If not can you suggest an alternative method? Would be nice to have this keyword-literal args documented to be able to figure out semantics.

Reply:
i think it is the author. to be sure though you can access the raw event using the `event` parameter: `event.comment.user` (see for example https://prod.repokitteh.app/traces/ui/envoyproxy/envoy/814a4880-32c4-11eb-8650-62ce98aed958).